network/book.go

```diff
@@ -177,42 +177,61 @@ func (s *SimpleBook) Failed(addr string) {
 	e.failure++
 }
 
-// Get will return the top priority peer out of all peers that are currently not active. It should
+// OrderedSample will return the top priority peer out of all peers that are currently not active. It should
 // represent the peer that is most likely to allow us to connect successfully.
-func (s *SimpleBook) Get() (string, error) {
+func (s *SimpleBook) OrderedSample(params ...int) ([]string, error) {
```

Comment:
Not a fan either, agree. I tried uint and len() complains if I try to apply == operator.